coap-gateway/service/signUp.go

```diff
@@ -153,6 +153,9 @@ func signOffHandler(req *mux.Message, client *Client) {
 	if deviceID == "" {
 		deviceID = authCurrentCtx.GetDeviceID()
 	}
+	if accessToken == "" {
+		accessToken = authCurrentCtx.GetAccessToken()
+	}
 
 	err := validateSignOff(deviceID, accessToken)
```

Comment:
Inside of validateSignOff is error message bit weird. If access token is empty, error message is "invalid  ". What is invalid?

Reply:
fixed